### ISSUE TITLE
Images for for Windows Server 2019 (1809)

### ISSUE THREAD

**Issue by pschichtel** [PowerShell/PowerShell-Docker#89]
Currently there doesn't seem to be a powershell docker images based on core or nano that is compatible with with Windows Server 2019 (1809) hosts. A nanoserver image is already available via `mcr.microsoft.com/windows/nanoserver:1809`.

**Comment by TravisEz13** [comment]
I'm waiting on an API to list the images from `mcr.microsoft.com`.  I asked them about this about an hour ago.

**Comment by TravisEz13** [comment]
Fixed in #111.  I will close the issue when the images are released.

**Comment by TravisEz13** [comment]
FYI, I'm having a build issue with nanoserver in Azure Container Registry (ACR) for 1809.  I've escalated to ACR.  In the meantime, I've submitted a PR to unblock the release of ServerCore 1809.

To build the nanoserver images locally run:
```
./build -Build -Channel stable -Name nanoserver -IncludeKnownIssues
```

`-IncludeKnowIssues` I'm currently adding.  The other channel is `preview` if you want the preview images.

When built like this, the nanoserver 1809 image will be called `powershell.local:6.1.1-nanoserver-1809`
